packages/client/src/__tests__/integration/happy/enums/schema.prisma

```diff
@@ -3,30 +3,30 @@ datasource my_db {
   url      = "file:dev.db"
 }
 
+model Location {
+  city String @default("")
+  id   Int    @id @default(autoincrement())
+  User User[]
+}
+
+model Post {
+  createdAt DateTime @default(dbgenerated("'1970-01-01 00:00:00'"))
+  email     String   @unique(map: "Post.email") @default("")
+  id        String   @id
+  name      String   @default("")
+  updatedAt DateTime @default(dbgenerated("'1970-01-01 00:00:00'"))
+  user      String?
+  User      User?    @relation(fields: [user], references: [id], onUpdate: NoAction)
+}
+
 model User {
-  email        String    @default("") @unique
+  email        String    @unique(map: "User.email") @default("")
```

Comment:
Is that "@unique(map: "User.email")" really expected? 🤔

Reply:
At least for this database, yes. 
Either the SQLite default naming scheme is different from what Prisma expects, or this database was created with these explicit names that do not match our naming scheme for unique indexes - so it renders them.